server/events.go

```diff
@@ -1673,7 +1673,9 @@ func (s *Server) debugSubscribers(sub *subscription, c *client, subject, reply s
 		}
 		// Cleanup the WC entry.
 		s.mu.Lock()
-		delete(s.sys.replies, replySubj)
+		if s.sys != nil && s.sys.replies != nil {
+			delete(s.sys.replies, replySubj)
+		}
```

Comment:
Not sure it makes sense to send the reply if we did not delete from the map, but I guess it does not hurt..

Reply:
True will fix..